src/main.js

```diff
@@ -208,17 +246,43 @@ const outputHTML = envSpecEntriesArray => {
     //return as string value
     return envSpecEntriesToPrint.join("");
   }
-  //in case of syntax error , print HTML format
-  return "Error:Wrong Syntax";
+  //in case of syntax error ,
+  //return "Error:Wrong Syntax";
 };
 
 /** @function envSpecToHTML
- * @desc gives final output using {@link outputHTML} and {@link checkValidationOfValues}
+ * @desc gives final output using {@link parse} , {@link outputHTML} and {@link checkValidationOfValues}
  * @param {string} envSpec the .env file given in string format
- * @returns {string} HTML code
+ * @returns {promise} that resolves to string
  */
 const envSpecToHTML = envSpec => {
-  return outputHTML(checkValidationOfValues(envSpec));
+  return parse(envSpec)
+    .then(function(entries) {
+      return entries.html();
+    })
+    .then(function(text) {
+      return text;
+    })
+    .catch(e => {
+      return e;
+    });
+};
+
+/** @function parse
+ * @desc transforms .env file to an array of valid values or an error message
+ * @param {string} envSpec the .env file given in string format
+ * @returns {promise} that resolves to an array of entry objets
+ */
+const parse = envSpecTxt => {
```

Comment:
Can you also export this function by adding the following code at the end of the file?

```js
module.exports.parse = parse;
```